### ISSUE TITLE
[AdaptiveCards- Schema Explorer]: In scan mode, Incorrect information is announced by the screen reader for the 'Feedback link'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8163]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description
**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-requisite:**
Turn on the narrator and turn on the scan mode.

**Repro Steps:**
1. Open above URL with valid credential.
2. Turn on the Narrator using ctrl+win+enter key.
3. Navigate to 'Schema Explorer' link and using arrow keys press enter key to activate it.
4. Navigate to 'Feedback' link using down arrow and observe the issue.

**Actual Result:**
While navigating to the 'Feedback' links in scan mode, Screen Reader does convey the irrelevant information. It is announcing as 'Enter list 1 of 2 level 1 bulleted list round bullets dark slate grey feedback'.

**Expected Result:**
Screen Reader should not convey the irrelevant information for the feedback link and it should announce as 'Link Feedback'.

**Note:**
Same Issue is repro with NVDA as well.

**User Experience:**
It will impact the screen reader user if the screen reader does convey the irrelevant information for the links.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9C514377-76E5-4CFE-8300-DA3079D3CDC3%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships.docx&action=default&mobileredirect=true&share=IQF3Q1Gc5Xb-TIMA2jB5083DAcWOZyaXcx7TtNeRlXM4uHk

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Card JSON

https://user-images.githubusercontent.com/95913748/208687460-b60257e4-a533-4846-baa1-c006a56c07b1.mp4

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
